### DISCUSSION TITLE
ART 1.12.1

### DISCUSSION THREAD

**Discussion by beat-buesser** [Trusted-AI/adversarial-robustness-toolbox#1862] [Announcements]
This release of ART 1.12.1 provides updates to ART 1.12.

# Added

[None]

# Changed

[None]

# Removed

[None] 

# Fixed

- Fixed object detection estimator `PyTorchYolo` to not modify tracked statistics of batch-norm layers of the YOLO model during loss and loss gradient calculations (#1860)

<hr /><em>This discussion was created from the release <a href='https://github.com/Trusted-AI/adversarial-robustness-toolbox/releases/tag/1.12.1'>ART 1.12.1</a>.</em>
